src/main/java/com/attorneyrequest/validation/ValidationWorkflow.java

```diff
@@ -0,0 +1,24 @@
+package com.attorneyrequest.validation;
+
+import com.attorneyrequest.common.Case;
+import com.attorneyrequest.data_entry.DataEntryWorkflow;
+
+public class ValidationWorkflow {
+    private static Case caseFile = null;
+    private static DataEntryWorkflow dataFlow = new DataEntryWorkflow();

```

Comment:
The dataFlow object is not correctly use in term of OOP manner. You don't need to create an object of itself and store it in the static variable of the same class. Suggestion, you might need a validateCaseFile() function which return either true or false. I'll merge this one for now.